org.eclipse.xtext.common.types/src/org/eclipse/xtext/common/types/util/AnnotationLookup.java

```diff
@@ -52,6 +52,19 @@ public JvmAnnotationReference findAnnotation(/* @NonNull */ JvmAnnotationTarget
 		return null;
 	}
 	
+	public JvmAnnotationReference findAnnotation(/* @NonNull */ JvmAnnotationTarget annotationTarget, /* @NonNull */ String lookupType) {
```

Comment:
Should the method above delegate to this implementation here?

Reply:
will change